src/drawer.js

```diff
@@ -479,6 +479,12 @@ $.Drawer.prototype = {
         if ( this.viewport.degrees !== 0 ) {
             this._offsetForRotation(this.viewport.degrees);
         }
+        if (tiledImage.degrees) {
```

Comment:
I think we want to encourage people using the API to use getRotation() rather than accessing degrees directly... of course this can be accomplished by not documenting degrees, but we could consider also marking it as private with an underscore: `_degrees`. What do you think?

Of course Drawer is a "friend" of TiledImage, so it's probably fine that it accesses the property directly (and it's presumably just a touch faster to do so).